docs/rule/no-action.md

```diff
@@ -4,7 +4,7 @@
 
 What's wrong with `{{action}}`?
 
-"Action" is an overloaded term in Ember parlance. Actions are:
```

Comment:
@judithhinlung It looks like you got a bad merge as there are a bunch of unrelated changes in this PR now.